practice-one/src/pages/Analytics/Analytics.jsx

```diff
@@ -1,7 +1,59 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import "./Analytics.css";
+import { Button, Expand, CustomerItem } from "@components";
+import { BUTTON_VARIANTS } from "@constants/buttons";
+import { plusIcon } from "@assets/images";
+import { EXPAND_TITLES } from "@data";
 
 const Analytics = () => {
-  return <h1>Analytic</h1>;
+  const [customers, setCustomers] = useState([]);
+
+  useEffect(() => {
+    // Sử dụng fetch để fetch dữ liệu từ API
+    fetch("https://64e70a09b0fd9648b78f3e0a.mockapi.io/customers")
+      .then((response) => response.json())
```

Comment:
create helper function

Reply:
I created services folder to call API
![image](https://github.com/datdragon777/React-Training/assets/74806826/e0f97681-47ed-48ee-a1c6-6d6a977faa85)

then I call function from service to Analytics component to get data
![image](https://github.com/datdragon777/React-Training/assets/74806826/b2ca5be0-f5b1-4c91-beb4-3a5abb28b255)